<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>list</class>
 <widget class="QWidget" name="list">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>播放列表</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
*:hover{
color: rgb(0, 170, 255);
}
#widget{
border-radius:15px;
background-color: rgb(0, 0, 0);
}
*{
font:100 12pt &quot;华文仿宋&quot;;
color: rgb(255, 255, 255);
background-color: transparent;
border:0px;
}
QLabel{
border-top:1px solid rgb(198, 198, 198);
border-bottom:1px solid rgb(198, 198, 198);
min-height:40px;
}
QLineEdit
{
border-bottom:1px solid rgb(198, 198, 198);
min-height:40px;
}
QComboBox{
min-height:40px;
}
QComboBox QAbstractItemView 
{
background-color: rgb(0, 0, 0);
}
QMenu
{
background-color: rgba(0,0,0,128);
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QPixmapShow" name="pixmapshow" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_show" native="true">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="pushButton_tables">
                 <property name="text">
                  <string>列表名</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_high">
                 <property name="text">
                  <string>高级设置</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QStackedWidget" name="stackedWidget_tables">
              <property name="currentIndex">
               <number>-1</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPixmapShow</class>
   <extends>QWidget</extends>
   <header location="global">qpixmapshow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
